src/System.Management.Automation/engine/MshCommandRuntime.cs

```diff
@@ -2819,7 +2819,7 @@ private void DoWriteError(object obj)
         /// but the command failure will ultimately be
         /// <see cref="System.Management.Automation.ActionPreferenceStopException"/>,
         /// </remarks>
-        internal void _WriteErrorSkipAllowCheck(ErrorRecord errorRecord, ActionPreference? actionPreference = null, bool isNativeError = false)
+        internal void _WriteErrorSkipAllowCheck(ErrorRecord errorRecord, ActionPreference? actionPreference = null, bool isNativeStdErrCall = false)
```

Comment:
Also, it would be helpful to add a xml comment for the parameters, to explain a bit more what it is and how it may be confusing to the error generated from non-zero exit code.